### ISSUE TITLE
Add support for Free Format parameter declaration

### ISSUE THREAD

**Issue by patrickhrastnik** [IBM/xmlservice#43]
Hi!

In [https://github.com/IBM/nodejs-itoolkit/issues/75](https://github.com/IBM/nodejs-itoolkit/issues/75) I suggested to add support for free format parameter syntax ("char(10)" instead of "10A").
@ThePrez suggested to open an issue here.

This may be something related to #24 - but as far as I've seen, #24 is more about changing this packages' source to total free syntax.

**Comment by jimoibm** [comment]
Agreed with what @kadler said in itoolkit#75. Current xmlservice is more than a low level component that is fledged with lots of features. Not sure how many of them are still used. 

Anyway supporting new format 'type' can be done in

https://github.com/IBM/xmlservice/blob/e9cfced8a9370f7e206f5e97d50b1452eaca15d4/src/plugile.rpgle#L2408

This procedure might have a if leg ahead to serve the new format to have below vars set up correctly ... 
https://github.com/IBM/xmlservice/blob/e9cfced8a9370f7e206f5e97d50b1452eaca15d4/src/plugile.rpgle#L2412-L2414

**Comment by kadler** [comment]
I don't think it makes much sense to invest in the format of the underlying transport format. How many users use XMLSERVICE directly vs using a toolkit which hides all this from the user?

**Comment by jimoibm** [comment]
Agree with @kadler again :-)
FWIW from customer issues, majorities are still in fixed format or mixed format. I will hold this and consider free format in future if this is really necessary.
